lib/Search/SearchQuery.php

```diff
@@ -154,7 +154,7 @@ private static function filterArray(array $array, $key, $default, $filter = null
      */
     public function getFrom()
     {
-        return $this->from;
+        return (int)$this->from;
```

Comment:
Unnecessary casting

Reply:
Just following the logic of phpdoc. Also this return value needed as an integer in calculation of pagination.

Reply:
Doesn't really matter, just pointing out that it will get removed when running inspection cleanup tools.

Reply:
Btw, It's handled at https://github.com/salesagility/SuiteCRM/blob/develop/modules/Home/Search.php#L52

Reply:
Then the inspection cleanup tool should ignore it as it's necessary.